### ISSUE TITLE
Steam Deck remote play 

### ISSUE THREAD

**Issue by MariusKlug** [araghon007/X1nput#45]
Hey,

I play dirt rally 2.0 remotely on the steam deck in the living room, connected via LAN to the PC running it, and having a TV and an Xbox elite series 2 controller connected (via Bluetooth). I am able to get the trigger rumble on the pc directly, but it seems to not get transferred when playing on the steam deck. Restarting everything and injecting into the game as before does not give me trigger haptics. 

I haven't seen a solution to this, but maybe it has to do with the fact that I need to inject into steam in that case? I will check that later, but wanted to have this issue here for future users as documentation.

**Comment by araghon007** [comment]
Yeah, this program can only modify the trigger rumble for controllers directly connected to the PC. 
I know for a fact that Valve's Xbox Controller driver does support trigger rumble, but they don't use that to ass rumble to games, only making it work for games that already have trigger rumble in the first place.

It might be possible to modify the trigger rumble values that Steam passes through Remote Play, but I never thought of doing that. Might give it a shot someday

**Comment by MariusKlug** [comment]
Ah snap. I was hoping it's the trivial thing of injecting into steam. Pity, I'll be playing mostly on the deck, not directly on the PC. The added latency is noticeable but the couch is really nice :D we'll see. Thanks for confirming! 

If you have a version to test for passing trigger rumble through remote play, let me know!
